src/audio/tdfb/tdfb_direction.c

```diff
@@ -125,10 +125,6 @@ static int16_t max_mic_distance(struct tdfb_comp_data *cd)
 
 	/* Max lag based on largest array dimension. Microphone coordinates are Q4.12 meters */
 	for (i = 0; i < cd->config->num_mic_locations; i++) {
-		for (j = 0; i < cd->config->num_mic_locations; i++) {
-			if (j == i)
-				continue;
```

Comment:
this breaks compilation and therefore bisection. No commit should break compilation.